protocol/validation/tx.go

```diff
@@ -227,7 +227,7 @@ func CheckTxWellFormed(tx *bc.Tx) error {
 			} else {
 				program = input.ControlProgram()
 			}
-			scriptStr, _ := vm.Disassemble(program)
+			scriptStr, _ := vm.Disassemble(program) // #nosec
```

Comment:
One option for at least making that case less confusing would be to check the error but not alter the rest of the execution, just print it out:

```
scriptStr, err := vm.Disassemble(program)
if err != nil {
	t.Log(err)
}
...
```